README.md

```diff
@@ -25,15 +25,15 @@
 
 > Describe your project in 1 or 2 sentences.
 
-**Portfolio: setup and mobile first** This is a project to strengthen our knowledge in HTML and CSS, but above all to put into practice the correct order of the elements through Flexbox.
+**Portfolio: setup and mobile first** This is a project to strengthen our knowledge in HTML and CSS, but above all to put into practice the correct order of the elements through Flexbox and Grip.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
 
-Html-Css-Flexbox-Figma.
+Html-Css-Flexbox-Grip-Figma.
 ### Key Features <a name="key-features"></a>
 
```

Comment:
@NahnahAJ hi, thank you very much for your review and your suggestions, Can optional changes requested by the code reviewer be omitted? Does this have a problem with the approval of the project? If I decide not to make all the optional changes at this time, will I have a problem with it by the next code reviewer?

Reply:
@Freherdez No it won’t affect the approval of your project. But you should make sure to implement them since they will help make your codebase better overall.